package.json

```diff
@@ -46,6 +46,7 @@
     "debug": "^4.1.1",
     "glob": "^7.1.6",
     "needle": "^2.5.0",
-    "tslib": "1.11.1"
+    "tmp": "^0.1.0",
```

Comment:
careful with these @magdziarek - this is re-introducing tmp dep because the PR was raised before we removed the tmp